src/sbvr-api/permissions.coffee

```diff
@@ -3,14 +3,16 @@ Promise = require 'bluebird'
 BluebirdLRU = require 'bluebird-lru-cache'
 env = require '../config-loader/env'
 userModel = require './user.sbvr'
-{ metadataEndpoints, BadRequestError, resolveSynonym } = require './uri-parser'
+{ metadataEndpoints, resolveSynonym } = require './uri-parser'
+{ BadRequestError } = require './errors'
 { ODataParser } = require '@resin/odata-parser'
 memoize = require 'memoizee'
 TypedError = require 'typed-error'
 sbvrUtils = require './sbvr-utils'
+{ PermissionError, PermissionParsingError } = require './errors'
```

Comment:
NIT: Can we join the two `error` requires to a single one here?

Reply:
Oh my... I really don't see how I missed that... Changing now